sdk/maps/Azure.Maps.Rendering/src/Models/Options/GetMapStaticImageOptions.cs

```diff
@@ -0,0 +1,91 @@
+        /// Longitude range: -180 to 180. Latitude range: -85 to 85.
+        /// Note: Either Center or BoundingBox are required parameters. They are mutually exclusive.
+        /// </summary>
+        public GeoPosition? CenterCoordinate { get; }
+        /// <summary>
+        /// Bounding box.
+        /// Note: Either BoundingBox or CenterCoordinate are required
+        /// parameters. They are mutually exclusive. It shouldn't be used with
+        /// HeightInPixels or WidthInPixels.
+        /// The maximum allowed ranges for Lat and Lon are defined for each zoom level
+        /// in the table at the top of this page.
+        /// </summary>
+        public GeoBoundingBox BoundingBox { get; }
+        /// <summary>
+        /// Height of the resulting image in pixels. Range is 1 to 8192. Default
+        /// is 512. It shouldn't be used with BoundingBox.
+        /// </summary>
+        public int? HeightInPixels { get; }
+        /// <summary> Width of the resulting image in pixels. Range is 1 to 8192. Default is 512. It shouldn't be used with bbox. </summary>
+        public int? WidthInPixels { get; }
+        /// <summary>
+        /// Language in which search results should be returned. Should be one of supported IETF language tags, case insensitive. When data in specified language is not available for a specific field, default language is used.
+        ///
+        /// Please refer to <see href="https://docs.microsoft.com/azure/azure-maps/supported-languages">Supported Languages</see> for details.
+        /// </summary>
+        public string RenderLanguage { get; set; }
+        /// <summary>
+        /// The View parameter (also called the &quot;user region&quot; parameter) allows you to show the correct maps for a certain country/region for geopolitically disputed regions. Different countries have different views of such regions, and the View parameter allows your application to comply with the view required by the country your application will be serving. By default, the View parameter is set to “Unified” even if you haven’t defined it in  the request. It is your responsibility to determine the location of your users, and then set the View parameter correctly for that location. Alternatively, you have the option to set ‘View=Auto’, which will return the map data based on the IP  address of the request. The View parameter in Azure Maps must be used in compliance with applicable laws, including those  regarding mapping, of the country where maps, images and other data and third party content that you are authorized to  access via Azure Maps is made available. Example: view=IN.
+        ///
+        /// Please refer to <see href="https://aka.ms/AzureMapsLocalizationViews">Supported Views</see> for details and to see the available Views.
+        /// </summary>
+        public Azure.Maps.LocalizedMapView? LocalizedMapView { get; set; }
+
+#pragma warning disable CA2227 // Collection properties should be read only
```

Comment:
@KrzysztofCwalina, @tg-msft:   Would you please offer your thoughts on the settable collection properties?

Reply:
To put more contents here. We make the settable collection because the lists are optional parameters. We don't want user to set the lists from ctor or create an empty list even when they are not used.

Reply:
I've approved the PR, but please resolve this with the architects.   Settable collection properties are contrary to guidelines, which is why there's an analyzer rule to enforce this constraint.

Reply:
I've removed pragma warning and make it a read only list.